### ISSUE TITLE
Spermatocyte arrest

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9956]
**Preferred term label:**
Spermatocyte arrest

**Synonyms**
Meiotic arrest

**Definition (free text, please give PubMed ID)**
Testicular histology shows seminiferous tubules with Sertoli cells, spermatogonial cells and spermatocytes but no further differentiated cells like round spermatids. 

**Parent term (use hpo.jax.org/app)**
HP:0031038

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
NA

**Your nano-attribution (ORCID)**
